ethcore/src/snapshot/tests/helpers.rs

```diff
@@ -51,10 +51,12 @@ impl StateProducer {
 		// modify existing accounts.
 		let mut accounts_to_modify: Vec<_> = {
 			let trie = TrieDB::new(&*db, &self.state_root).unwrap();
-			trie.iter()
+			let regionck_bug = trie.iter()
```

Comment:
right - figured it might be something like that. shame it's buggy. if there's a tracking issue in the rust GH would be good to note it so can be fixed later.